pkg/operator/controller/controller_test.go

```diff
@@ -520,6 +1222,34 @@ func getDeployment(client realClient.Client, deployment *appsv1.Deployment) (*ap
 	return result.(*appsv1.Deployment), nil
 }
 
+func equalObjects(a, b runtime.Object) (bool, error) {
```

Comment:
is this used/necessary?  what's wrong with `reflect.DeepEqual`?

Reply:
nothing wrong (was not aware of that) - fixed